t3f/tensor_train_base.py

```diff
@@ -151,6 +151,28 @@ def __add__(self, other):
     from t3f import ops
     return ops.add(self, other)
 
+  def __sub__(self, other):
```

Comment:
Its a bad practice to pack lots of unrelated stuff in one pull request. Its ok for now, but please avoid in the future (e.g. you could split this in 3 pull requests)